dll/win32/syssetup/wizard.c

```diff
@@ -2968,6 +2970,29 @@ ProcessSetupInf(
     }
 #endif
 
+    *szValue = UNICODE_NULL;
+    pData = szValue;
+
+    GetWindowsDirectoryW(szValue, ARRAYSIZE(szValue));
+    StringCchCatW(szValue, ARRAYSIZE(szValue), L"\\inf");
+
+    if (FAILED(StringCchLengthW(szValue, ARRAYSIZE(szValue), &cchLength)))
```

Comment:
Since you are so careful as to check the returned values of StringCchLengthW for errors, shouldn't you also check whether StringCchCatW above and StringCchCopyW below succeed? (in both cases, truncated versions of the strings concatenated or copied will be stored and NULL-terminated, but those other StringCchLengthW calls will succeed)